esponse text = {"errorMessages":[],"errors":{"issuetype":"Especifica un tipo de incidencia v??lido"}}
2023-05-12T15:44:14.255389878Z: [INFO]  OCurrio un error en la ejecuci??n de crear requerimiento: local variable 'newIssue' referenced before assignment
2023-05-12T15:44:14.255393816Z: [INFO]  169.254.129.1 - - [12/May/2023:15:44:14 +0000] "POST /CreateIssue HTTP/1.1" 500 265 "-" "PostmanRuntime/7.32.2"


[0:49 p. m., 12/5/2023] Jony Prome: 2023-05-12T15:48:56  Welcome, you are now connected to log-streaming service.

Starting Log Tail -n 10 of existing logs ----

/appsvctmp/volatile/logs/runtime/943a1e1a60f1938729047ffe19e13c5caa05e94d409c51ab5f7f00b442ea0d20.log 
2023-05-12T15:44:14.255311080Z: [INFO]  Requerimiento encontrado: 324
2023-05-12T15:44:14.255323253Z: [INFO]  325
2023-05-12T15:44:14.255328272Z: [INFO]  None
2023-05-12T15:44:14.255332550Z: [INFO]  data en funcion {'summary': 'Prueba carga requerimiento pre producci??n', 'key': 'GDD', 'description': 'completar', 'type': 'Tarea', 'approvers': 'Juan Carlos Canepa', 'managment': 'algo', 'impact': " + '\n Enlace a la Documentaci??n: ", 'attached': '', 'priority': '1', 'initiative': 'iniciativa'}
2023-05-12T15:44:14.255338231Z: [INFO]  E…
[0:49 p. m., 12/5/2023] Jony Prome: es lo unico que hay
[0:50 p. m., 12/5/2023] max: que raro
[0:50 p. m., 12/5/2023] max: y lo que staba antes se borro?
[0:50 p. m., 12/5/2023] Jony Prome: Y si
[0:50 p. m., 12/5/2023] Jony Prome: Siempre que haces un push
[0:50 p. m., 12/5/2023] Jony Prome: Sino se hace un log infinito
[0:50 p. m., 12/5/2023] max: y ahora no salio
[0:50 p. m., 12/5/2023] max: ah si mira la ultima linea
[0:50 p. m., 12/5/2023] max: pero faltan coasas
[0:51 p. m., 12/5/2023] max: me sirve
[2:33 p. m., 12/5/2023] max: otro log -.-
[2:33 p. m., 12/5/2023] max: por favor
[2:33 p. m., 12/5/2023] Jony Prome: despues de 97 min sin actividad
[2:33 p. m., 12/5/2023] Jony Prome: 2023-05-12T17:26:17.299343584Z: [INFO]  169.254.129.1 - - [12/May/2023:17:26:17 +0000] "GET / HTTP/1.1" 200 10943 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"

2023-05-12T17:26:17.556411781Z: [INFO]  169.254.129.1 - - [12/May/2023:17:26:17 +0000] "GET /static/css/style.css HTTP/1.1" 200 0 "https://gdrback.azurewebsites.net/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"

2023-05-12T17:26:17.939623658Z: [INFO]  169.254.129.1 - - [12/May/2023:17:26:17 +0000] "GET /static/assets/renderGestor.png HTTP/1.1" 200 0 "https://gdrback.azurewebsites.net/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"

2023-05-12T17:26:17.946041425Z: [INFO]  169.254.129.1 - - [12/May/2023:17:26:17 +0000] "GET /static/assets/triangulosbackII@4x.png HTTP/1.1" 200 0 "https://gdrback.azurewebsites.net/static/css/style.css" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"

2023-05-12T17:26:51.128332070Z: [INFO]  169.254.129.1 - - [12/May/2023:17:26:51 +0000] "GET /static/assets/favicon.ico HTTP/1.1" 200 0 "https://gdrback.azurewebsites.net/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"

2023-05-12T17:27:33.607820758Z: [INFO]     ___                               

2023-05-12T17:27:33.607849984Z: [INFO]    /  _  \ ____ ___   __  

2023-05-12T17:27:33.607855104Z: [INFO]   /  /\  \\_   /  |  \  _ \/ __ \ 

2023-05-12T17:27:33.607859412Z: [INFO]  /    |    \/    /|  |  /|  | \/\  _/ 

2023-05-12T17:27:33.607863260Z: [INFO]  \__|_  /__ \__/ |_|    \__  >

2023-05-12T17:27:33.607867228Z: [INFO]          \/      \/                  \/ 

2023-05-12T17:27:33.607870925Z: [INFO]  A P P   S E R V I C E   O N   L I N U X

2023-05-12T17:27:33.607874331Z: [INFO]  

2023-05-12T17:27:33.607955338Z: [INFO]  Documentation: http://aka.ms/webapp-linux

2023-05-12T17:27:33.607963284Z: [INFO]  Python 3.10.10

2023-05-12T17:27:33.607967522Z: [INFO]  Note: Any data outside '/home' is not persisted

2023-05-12T17:27:34.295924067Z: [INFO]  Starting OpenBSD Secure Shell server: sshd.

2023-05-12T17:27:34.353718483Z: [INFO]  App Command Line not configured, will attempt auto-detect

2023-05-12T17:27:34.470223112Z: [INFO]  Starting periodic command scheduler: cron.

2023-05-12T17:27:34.470745103Z: [INFO]  Launching oryx with: create-script -appPath /home/site/wwwroot -output /opt/startup/startup.sh -virtualEnvName antenv -defaultApp /opt/defaultsite

2023-05-12T17:27:34.517344009Z: [INFO]  Found build manifest file at '/home/site/wwwroot/oryx-manifest.toml'. Deserializing it...

2023-05-12T17:27:34.519824813Z: [INFO]  Build Operation ID: a03ce67d803aa6ca

2023-05-12T17:27:34.520848398Z: [INFO]  Oryx Version: 0.2.20230322.1, Commit: 6c04ec75685b441c9378c238b6491cf9812313db, ReleaseTagName: 20230322.1

2023-05-12T17:27:34.520882454Z: [ERROR]  Output is compressed. Extracting it...

2023-05-12T17:27:34.533056160Z: [ERROR]  Extracting '/home/site/wwwroot/output.tar.gz' to directory '/tmp/8db530e0bf8ec27'...

2023-05-12T17:27:35.565775873Z: [ERROR]  App path is set to '/tmp/8db530e0bf8ec27'

2023-05-12T17:27:36.285786871Z: [ERROR]  Detected an app based on Flask

2023-05-12T17:27:36.285919337Z: [ERROR]  Generating `gunicorn` command for 'app:app'

2023-05-12T17:27:36.335015838Z: [INFO]  Writing output script to '/opt/startup/startup.sh'

2023-05-12T17:27:36.562716045Z: [INFO]  Using packages from virtual environment antenv located at /tmp/8db530e0bf8ec27/antenv.

2023-05-12T17:27:36.563279236Z: [INFO]  Updated PYTHONPATH to '/opt/startup/app_logs:/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages'

2023-05-12T17:27:37.214005770Z: [ERROR]  [2023-05-12 17:27:37 +0000] [70] [INFO] Starting gunicorn 20.1.0

2023-05-12T17:27:37.235556076Z: [ERROR]  [2023-05-12 17:27:37 +0000] [70] [INFO] Listening at: http://0.0.0.0:8000 (70)

2023-05-12T17:27:37.235576325Z: [ERROR]  [2023-05-12 17:27:37 +0000] [70] [INFO] Using worker: sync

2023-05-12T17:27:37.237860849Z: [ERROR]  [2023-05-12 17:27:37 +0000] [73] [INFO] Booting worker with pid: 73

2023-05-12T17:27:39.142275955Z: [INFO]  169.254.129.1 - - [12/May/2023:17:27:39 +0000] "GET /robots933456.txt HTTP/1.1" 404 207 "-" "HealthCheck/1.0"

2023-05-12T17:28:57  No new trace in the past 1 min(s).
2023-05-12T17:29:57  No new trace in the past 2 min(s).
2023-05-12T17:30:57  No new trace in the past 3 min(s).
2023-05-12T17:31:57  No new trace in the past 4 min(s).
2023-05-12T17:32:07.510907491Z: [INFO]  169.254.129.1 - - [12/May/2023:17:32:07 +0000] "GET / HTTP/1.1" 200 10943 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"

2023-05-12T17:32:07.705666767Z: [INFO]  169.254.129.1 - - [12/May/2023:17:32:07 +0000] "GET /static/css/style.css HTTP/1.1" 200 0 "https://gdrback.azurewebsites.net/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"

2023-05-12T17:32:07.879623804Z: [INFO]  169.254.129.1 - - [12/May/2023:17:32:07 +0000] "GET /static/assets/renderGestor.png HTTP/1.1" 200 0 "https://gdrback.azurewebsites.net/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"

2023-05-12T17:32:07.943326750Z: [INFO]  169.254.129.1 - - [12/May/2023:17:32:07 +0000] "GET /static/assets/triangulosbackII@4x.png HTTP/1.1" 200 0 "https://gdrback.azurewebsites.net/static/css/style.css" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"

2023-05-12T17:32:14.985627727Z: [INFO]  esto es getallprojects controllers

2023-05-12T17:32:14.985668714Z: [INFO]  [{'key': 'GP0007', 'name': 'Nuevo workflow de colegiatura'}, {'key': 'SP000BN', 'name': 'Score prome '}, {'key': 'RDG', 'name': 'RendiciÃ³n de gastos'}, {'key': 'GDD', 'name': 'GestiÃ³n de la demanda'}, {'key': 'GT', 'name': 'GestiÃ³n de tecnologÃ­a'}]

2023-05-12T17:32:14.985677430Z: [INFO]  169.254.129.1 - - [12/May/2023:17:32:14 +0000] "GET /GetAllProjects HTTP/1.1" 200 263 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"

2023-05-12T17:32:26.768693868Z: [ERROR]  [2023-05-12 17:32:26,765] ERROR in app: Exception on /CreateIssue [POST]

2023-05-12T17:32:26.768749923Z: [ERROR]  Traceback (most recent call last):

2023-05-12T17:32:26.768757888Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask/app.py", line 2525, in wsgi_app

2023-05-12T17:32:26.768763609Z: [ERROR]      response = self.full_dispatch_request()

2023-05-12T17:32:26.768769029Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask/app.py", line 1822, in full_dispatch_request

2023-05-12T17:32:26.768773928Z: [ERROR]      rv = self.handle_user_exception(e)

2023-05-12T17:32:26.768778226Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask_cors/extension.py", line 165, in wrapped_function

2023-05-12T17:32:26.768782915Z: [ERROR]      return cors_after_request(app.make_response(f(*args, **kwargs)))

2023-05-12T17:32:26.768815135Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask/app.py", line 1820, in full_dispatch_request

2023-05-12T17:32:26.768821126Z: [ERROR]      rv = self.dispatch_request()

2023-05-12T17:32:26.768825665Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask/app.py", line 1796, in dispatch_request

2023-05-12T17:32:26.768829793Z: [ERROR]      return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)

2023-05-12T17:32:26.768833920Z: [ERROR]    File "/tmp/8db530e0bf8ec27/source/jiraModule/components/createIssue/view_createIssue.py", line 26, in CreateNewIssue

2023-05-12T17:32:26.768838419Z: [ERROR]      response = controller_createIssue.createIssue(dataIssue)

2023-05-12T17:32:26.768842587Z: [ERROR]    File "/tmp/8db530e0bf8ec27/source/jiraModule/components/createIssue/controller_createIssue.py", line 151, in createIssue

2023-05-12T17:32:26.768859729Z: [ERROR]      return jsonify({"link":link, "key":newIssue.key})

2023-05-12T17:32:26.768863776Z: [ERROR]  UnboundLocalError: local variable 'link' referenced before assignment

2023-05-12T17:32:26.770816262Z: [INFO]  No se encontrÃ³ el nÃºmero de requerimiento en el campo 'summary'.

2023-05-12T17:32:26.770838634Z: [INFO]  Requerimiento encontrado: 324

2023-05-12T17:32:26.770844906Z: [INFO]  Este es el id del ultimo requerimiento: 325

2023-05-12T17:32:26.770849604Z: [INFO]  None

2023-05-12T17:32:26.770854504Z: [INFO]  data en funcion {'summary': 'Prueba carga requerimiento pre producciÃ³n', 'key': 'GDD', 'description': 'completar', 'type': 'Tarea', 'approvers': 'Juan Carlos Canepa', 'managment': 'algo', 'impact': " + '\n Enlace a la DocumentaciÃ³n: ", 'attached': '', 'priority': '1', 'initiative': 'iniciativa'}

2023-05-12T17:32:26.770860235Z: [INFO]  Error al crear el issue en JIRA: JiraError HTTP 400 url: https://provinciamicroempresas.atlassian.net/rest/api/2/issue

2023-05-12T17:32:26.770864733Z: [INFO]  	

2023-05-12T17:32:26.770869692Z: [INFO]  	response headers = {'Date': 'Fri, 12 May 2023 17:32:26 GMT', 'Content-Type': 'application/json;charset=UTF-8', 'Server': 'AtlassianEdge', 'Timing-Allow-Origin': '*', 'X-Arequestid': '99e6065657b7b99dee8606b868bf55f4', 'X-Aaccountid': '6228d8734160640069ca5686', 'Cache-Control': 'no-cache, no-store, no-transform', 'X-Content-Type-Options': 'nosniff', 'X-Xss-Protection': '1; mode=block', 'Atl-Traceid': '8de0c958996bffda', 'Report-To': '{"endpoints": [{"url": "https://dz8aopenkvv6s.cloudfront.net"}], "group": "endpoint-1", "include_subdomains": true, "max_age": 600}', 'Nel': '{"failure_fraction": 0.001, "include_subdomains": true, "max_age": 600, "report_to": "endpoint-1"}', 'Strict-Transport-Security': 'max-age=63072000; includeSubDomains; preload', 'Transfer-Encoding': 'chunked'}

2023-05-12T17:32:26.770880953Z: [INFO]  	response text = {"errorMessages":[],"errors":{"issuetype":"Especifica un tipo de incidencia vÃ¡lido"}}

2023-05-12T17:32:26.770885912Z: [INFO]  Ocurrio un error en la ejecucion de crear requerimiento: local variable 'newIssue' referenced before assignment

2023-05-12T17:32:26.770889840Z: [INFO]  169.254.129.1 - - [12/May/2023:17:32:26 +0000] "POST /CreateIssue HTTP/1.1" 500 265 "-" "PostmanRuntime/7.32.2"

2023-05-12T17:32:31.529415789Z: [ERROR]  [2023-05-12 17:32:31,528] ERROR in app: Exception on /CreateIssue [POST]

2023-05-12T17:32:31.529473717Z: [ERROR]  Traceback (most recent call last):

2023-05-12T17:32:31.529480550Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask/app.py", line 2525, in wsgi_app

2023-05-12T17:32:31.529485510Z: [ERROR]      response = self.full_dispatch_request()

2023-05-12T17:32:31.529489647Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask/app.py", line 1822, in full_dispatch_request

2023-05-12T17:32:31.529493865Z: [ERROR]      rv = self.handle_user_exception(e)

2023-05-12T17:32:31.529497752Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask_cors/extension.py", line 165, in wrapped_function

2023-05-12T17:32:31.529502020Z: [ERROR]      return cors_after_request(app.make_response(f(*args, **kwargs)))

2023-05-12T17:32:31.529505998Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask/app.py", line 1820, in full_dispatch_request

2023-05-12T17:32:31.529523220Z: [ERROR]      rv = self.dispatch_request()

2023-05-12T17:32:31.529527027Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask/app.py", line 1796, in dispatch_request

2023-05-12T17:32:31.529530724Z: [ERROR]      return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)

2023-05-12T17:32:31.529534281Z: [ERROR]    File "/tmp/8db530e0bf8ec27/source/jiraModule/components/createIssue/view_createIssue.py", line 26, in CreateNewIssue

2023-05-12T17:32:31.529538338Z: [ERROR]      response = controller_createIssue.createIssue(dataIssue)

2023-05-12T17:32:31.529542496Z: [ERROR]    File "/tmp/8db530e0bf8ec27/source/jiraModule/components/createIssue/controller_createIssue.py", line 151, in createIssue

2023-05-12T17:32:31.529546534Z: [ERROR]      return jsonify({"link":link, "key":newIssue.key})

2023-05-12T17:32:31.529550772Z: [ERROR]  UnboundLocalError: local variable 'link' referenced before assignment

2023-05-12T17:32:31.530890484Z: [INFO]  No se encontrÃ³ el nÃºmero de requerimiento en el campo 'summary'.

2023-05-12T17:32:31.530903298Z: [INFO]  Requerimiento encontrado: 324

2023-05-12T17:32:31.530907996Z: [INFO]  Este es el id del ultimo requerimiento: 325

2023-05-12T17:32:31.530911934Z: [INFO]  None

2023-05-12T17:32:31.530915871Z: [INFO]  data en funcion {'summary': 'Prueba carga requerimiento pre producciÃ³n', 'key': 'GDD', 'description': 'completar', 'type': 'Tarea', 'approvers': 'Juan Carlos Canepa', 'managment': 'algo', 'impact': " + '\n Enlace a la DocumentaciÃ³n: ", 'attached': '', 'priority': '1', 'initiative': 'iniciativa'}

2023-05-12T17:32:31.530920560Z: [INFO]  Error al crear el issue en JIRA: JiraError HTTP 400 url: https://provinciamicroempresas.atlassian.net/rest/api/2/issue

2023-05-12T17:32:31.530924668Z: [INFO]  	

2023-05-12T17:32:31.530949885Z: [INFO]  	response headers = {'Date': 'Fri, 12 May 2023 17:32:31 GMT', 'Content-Type': 'application/json;charset=UTF-8', 'Server': 'AtlassianEdge', 'Timing-Allow-Origin': '*', 'X-Arequestid': '8b1cc6961087c8b347760e021e7065f7', 'X-Aaccountid': '6228d8734160640069ca5686', 'Cache-Control': 'no-cache, no-store, no-transform', 'X-Content-Type-Options': 'nosniff', 'X-Xss-Protection': '1; mode=block', 'Atl-Traceid': '8c7d73026a0c6f39', 'Report-To': '{"endpoints": [{"url": "https://dz8aopenkvv6s.cloudfront.net"}], "group": "endpoint-1", "include_subdomains": true, "max_age": 600}', 'Nel': '{"failure_fraction": 0.001, "include_subdomains": true, "max_age": 600, "report_to": "endpoint-1"}', 'Strict-Transport-Security': 'max-age=63072000; includeSubDomains; preload', 'Transfer-Encoding': 'chunked'}

2023-05-12T17:32:31.530957900Z: [INFO]  	response text = {"errorMessages":[],"errors":{"issuetype":"Especifica un tipo de incidencia vÃ¡lido"}}

2023-05-12T17:32:31.530962158Z: [INFO]  Ocurrio un error en la ejecucion de crear requerimiento: local variable 'newIssue' referenced before assignment

2023-05-12T17:32:31.530966205Z: [INFO]  169.254.129.1 - - [12/May/2023:17:32:31 +0000] "POST /CreateIssue HTTP/1.1" 500 265 "-" "PostmanRuntime/7.32.2"

2023-05-12T17:32:35.396150814Z: [ERROR]  [2023-05-12 17:32:35,395] ERROR in app: Exception on /CreateIssue [POST]

2023-05-12T17:32:35.396197731Z: [ERROR]  Traceback (most recent call last):

2023-05-12T17:32:35.396205376Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask/app.py", line 2525, in wsgi_app

2023-05-12T17:32:35.396333596Z: [ERROR]      response = self.full_dispatch_request()

2023-05-12T17:32:35.396338826Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask/app.py", line 1822, in full_dispatch_request

2023-05-12T17:32:35.396342953Z: [ERROR]      rv = self.handle_user_exception(e)

2023-05-12T17:32:35.396346520Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask_cors/extension.py", line 165, in wrapped_function

2023-05-12T17:32:35.396350517Z: [ERROR]      return cors_after_request(app.make_response(f(*args, **kwargs)))

2023-05-12T17:32:35.396354325Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask/app.py", line 1820, in full_dispatch_request

2023-05-12T17:32:35.396358011Z: [ERROR]      rv = self.dispatch_request()

2023-05-12T17:32:35.396363672Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask/app.py", line 1796, in dispatch_request

2023-05-12T17:32:35.396367589Z: [ERROR]      return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)

2023-05-12T17:32:35.396371176Z: [ERROR]    File "/tmp/8db530e0bf8ec27/source/jiraModule/components/createIssue/view_createIssue.py", line 26, in CreateNewIssue

2023-05-12T17:32:35.396374873Z: [ERROR]      response = controller_createIssue.createIssue(dataIssue)

2023-05-12T17:32:35.396378220Z: [ERROR]    File "/tmp/8db530e0bf8ec27/source/jiraModule/components/createIssue/controller_createIssue.py", line 151, in createIssue

2023-05-12T17:32:35.396382187Z: [ERROR]      return jsonify({"link":link, "key":newIssue.key})

2023-05-12T17:32:35.396386194Z: [ERROR]  UnboundLocalError: local variable 'link' referenced before assignment

2023-05-12T17:32:35.399359004Z: [INFO]  No se encontrÃ³ el nÃºmero de requerimiento en el campo 'summary'.

2023-05-12T17:32:35.399383129Z: [INFO]  Requerimiento encontrado: 324

2023-05-12T17:32:35.399389200Z: [INFO]  Este es el id del ultimo requerimiento: 325

2023-05-12T17:32:35.399393468Z: [INFO]  None

2023-05-12T17:32:35.399397556Z: [INFO]  data en funcion {'summary': 'Prueba carga requerimiento pre producciÃ³n', 'key': 'GDD', 'description': 'completar', 'type': 'Tarea', 'approvers': 'Juan Carlos Canepa', 'managment': 'algo', 'impact': " + '\n Enlace a la DocumentaciÃ³n: ", 'attached': '', 'priority': '1', 'initiative': 'iniciativa'}

2023-05-12T17:32:35.399402686Z: [INFO]  Error al crear el issue en JIRA: JiraError HTTP 400 url: https://provinciamicroempresas.atlassian.net/rest/api/2/issue

2023-05-12T17:32:35.399406843Z: [INFO]  	

2023-05-12T17:32:35.399410831Z: [INFO]  	response headers = {'Date': 'Fri, 12 May 2023 17:32:35 GMT', 'Content-Type': 'application/json;charset=UTF-8', 'Server': 'AtlassianEdge', 'Timing-Allow-Origin': '*', 'X-Arequestid': 'c11b2d816fe57a40affca79fff801ed9', 'X-Aaccountid': '6228d8734160640069ca5686', 'Cache-Control': 'no-cache, no-store, no-transform', 'X-Content-Type-Options': 'nosniff', 'X-Xss-Protection': '1; mode=block', 'Atl-Traceid': '946fca18124f5d52', 'Report-To': '{"endpoints": [{"url": "https://dz8aopenkvv6s.cloudfront.net"}], "group": "endpoint-1", "include_subdomains": true, "max_age": 600}', 'Nel': '{"failure_fraction": 0.001, "include_subdomains": true, "max_age": 600, "report_to": "endpoint-1"}', 'Strict-Transport-Security': 'max-age=63072000; includeSubDomains; preload', 'Transfer-Encoding': 'chunked'}

2023-05-12T17:32:35.399428013Z: [INFO]  	response text = {"errorMessages":[],"errors":{"issuetype":"Especifica un tipo de incidencia vÃ¡lido"}}

2023-05-12T17:32:35.399434134Z: [INFO]  Ocurrio un error en la ejecucion de crear requerimiento: local variable 'newIssue' referenced before assignment

2023-05-12T17:32:35.399439204Z: [INFO]  169.254.129.1 - - [12/May/2023:17:32:35 +0000] "POST /CreateIssue HTTP/1.1" 500 265 "-" "PostmanRuntime/7.32.2"

2023-05-12T17:32:55.468932410Z: [ERROR]  [2023-05-12 17:32:55,468] ERROR in app: Exception on /CreateIssue [POST]

2023-05-12T17:32:55.468999215Z: [ERROR]  Traceback (most recent call last):

2023-05-12T17:32:55.469005226Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask/app.py", line 2525, in wsgi_app

2023-05-12T17:32:55.469009935Z: [ERROR]      response = self.full_dispatch_request()

2023-05-12T17:32:55.469013502Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask/app.py", line 1822, in full_dispatch_request

2023-05-12T17:32:55.469017749Z: [ERROR]      rv = self.handle_user_exception(e)

2023-05-12T17:32:55.469021667Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask_cors/extension.py", line 165, in wrapped_function

2023-05-12T17:32:55.469025885Z: [ERROR]      return cors_after_request(app.make_response(f(*args, **kwargs)))

2023-05-12T17:32:55.469029842Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask/app.py", line 1820, in full_dispatch_request

2023-05-12T17:32:55.469034010Z: [ERROR]      rv = self.dispatch_request()

2023-05-12T17:32:55.469037777Z: [ERROR]    File "/tmp/8db530e0bf8ec27/antenv/lib/python3.10/site-packages/flask/app.py", line 1796, in dispatch_request

2023-05-12T17:32:55.469042055Z: [ERROR]      return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)

2023-05-12T17:32:55.469046493Z: [ERROR]    File "/tmp/8db530e0bf8ec27/source/jiraModule/components/createIssue/view_createIssue.py", line 26, in CreateNewIssue

2023-05-12T17:32:55.469050992Z: [ERROR]      response = controller_createIssue.createIssue(dataIssue)

2023-05-12T17:32:55.469055340Z: [ERROR]    File "/tmp/8db530e0bf8ec27/source/jiraModule/components/createIssue/controller_createIssue.py", line 151, in createIssue

2023-05-12T17:32:55.469059397Z: [ERROR]      return jsonify({"link":link, "key":newIssue.key})

2023-05-12T17:32:55.469063685Z: [ERROR]  UnboundLocalError: local variable 'link' referenced before assignment

2023-05-12T17:32:55.470761576Z: [INFO]  No se encontrÃ³ el nÃºmero de requerimiento en el campo 'summary'.

2023-05-12T17:32:55.470788426Z: [INFO]  Requerimiento encontrado: 324

2023-05-12T17:32:55.470795199Z: [INFO]  Este es el id del ultimo requerimiento: 325

2023-05-12T17:32:55.470799076Z: [INFO]  None

2023-05-12T17:32:55.470803184Z: [INFO]  data en funcion {'summary': 'Prueba carga requerimiento pre producciÃ³n', 'key': 'GDD', 'description': 'completar', 'type': 'Tarea', 'approvers': 'Juan Carlos Canepa', 'managment': 'algo', 'impact': " + '\n Enlace a la DocumentaciÃ³n: ", 'attached': '', 'priority': '1', 'initiative': 'iniciativa'}

2023-05-12T17:32:55.470808293Z: [INFO]  Error al crear el issue en JIRA: JiraError HTTP 400 url: https://provinciamicroempresas.atlassian.net/rest/api/2/issue

2023-05-12T17:32:55.470827469Z: [INFO]  	

2023-05-12T17:32:55.470832970Z: [INFO]  	response headers = {'Date': 'Fri, 12 May 2023 17:32:55 GMT', 'Content-Type': 'application/json;charset=UTF-8', 'Server': 'AtlassianEdge', 'Timing-Allow-Origin': '*', 'X-Arequestid': 'a31639de9a5b77fcd28f9c6fb6ec4086', 'X-Aaccountid': '6228d8734160640069ca5686', 'Cache-Control': 'no-cache, no-store, no-transform', 'X-Content-Type-Options': 'nosniff', 'X-Xss-Protection': '1; mode=block', 'Atl-Traceid': 'b8106b075e55f997', 'Report-To': '{"endpoints": [{"url": "https://dz8aopenkvv6s.cloudfront.net"}], "group": "endpoint-1", "include_subdomains": true, "max_age": 600}', 'Nel': '{"failure_fraction": 0.001, "include_subdomains": true, "max_age": 600, "report_to": "endpoint-1"}', 'Strict-Transport-Security': 'max-age=63072000; includeSubDomains; preload', 'Transfer-Encoding': 'chunked'}

2023-05-12T17:32:55.470840534Z: [INFO]  	response text = {"errorMessages":[],"errors":{"issuetype":"Especifica un tipo de incidencia vÃ¡lido"}}

2023-05-12T17:32:55.470845242Z: [INFO]  Ocurrio un error en la ejecucion de crear requerimiento: local variable 'newIssue' referenced before assignment

2023-05-12T17:32:55.470849070Z: [INFO]  169.254.129.1 - - [12/May/2023:17:32:55 +0000] "POST /CreateIssue HTTP/1.1" 500 265 "-" "PostmanRuntime/7.32.2"

2023-05-12T17:33:14.937013037Z: [INFO]  esto es getallprojects controllers

2023-05-12T17:33:14.937055136Z: [INFO]  [{'key': 'GP0007', 'name': 'Nuevo workflow de colegiatura'}, {'key': 'SP000BN', 'name': 'Score prome '}, {'key': 'RDG', 'name': 'RendiciÃ³n de gastos'}, {'key': 'GDD', 'name': 'GestiÃ³n de la demanda'}, {'key': 'GT', 'name': 'GestiÃ³n de tecnologÃ­a'}]

2023-05-12T17:33:14.937064122Z: [INFO]  169.254.129.1 - - [12/May/2023:17:33:14 +0000] "GET /GetAllProjects HTTP/1.1" 200 263 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"